.github/workflows/go.unit.tests.yaml

```diff
@@ -20,6 +20,7 @@ name: Go Unit Tests
 on:
   push:
     paths:
+      - GO_VERSION
```

Comment:
Shouldn't this also be added to the go-fmt action?

Reply:
Yes. Added in 7c14c75271